README.md

```diff
@@ -90,7 +90,7 @@ Git actually destroys the old commits and replaces them with new commits
 
 This is one of the things that can make `git rebase` dangerous,
  and it's the reason why you never rebase code that's already been
- published and shared - you run the risk of breaking other peoples' code.
+ published and shared - you run the risk of breaking other peoples' code. After a rebase, pushing to the branch will require --force-with-lease since the history has been rewritten.
```

Comment:
Actually, this is included below. Remove the 'After a rebase code here.' I certainly don't want to reinforce the -force stuff twice. So remove it here, keep it below.